my-app/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,27 @@
+import React, {useState} from 'react'
+import "./style.css"
+import BurgerMenu from '../BurgerMenu'
+
+const Header = () => {
+   const [isOpen, setIsOpen] = useState(false);
+   const handleBurgerClick = () => {
+      setIsOpen(!isOpen);
+   }
+   return (
+      <div className='header__wrapper'>
+         <BurgerMenu customClass='burger' text={isOpen ? 'x' : '☰'} onClick={handleBurgerClick}>
```

Comment:
Alternative way to do this:

`<BurgerMenu customClass='burger' text={isOpen ? 'x' : '☰'} onClick={() => setIsOpen(!isOpen)}>`